skills/debug/references/thorough-track.md

```
@@ -0,0 +1,171 @@
+# Thorough Track
+
+## Purpose
+
+Fix bugs with proper rigor. Capture institutional knowledge through RCA.
+
+## Phases
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)